bluebrain/deployment/environments/applications_hpc.yaml

```
@@ -7,6 +7,7 @@ spack:
           - asciitoh5
           - model-neocortex
           - nest
+          - neurodamus  # experimental
```

```suggestion
          - neurodamus
```
No one is going to see that comment.